### PR TITLE
NO-ISSUE Support custom data coding

### LINKED ISSUE #64: Setting DCS explicitly
I am trying to set the data coding scheme to 0xf6(246) to test SAT push messaging in a production environment. I couldn't find a way to provision the DCS explicitly for this. Could anyone please help here.. Thank you very much.

### PR DESCRIPTION
**Pull request by linxGnu**
Closes #64

### CODE REVIEW

File: data/codings.go [hunk 1]
```diff
@@ -49,25 +49,52 @@ func decode(data []byte, decoder *encoding.Decoder) (st string, err error) {
...
+
+func NewCustomEncoding(coding byte, encDec EncDec) Encoding {
+	return &CustomEncoding{
+		coding: coding,
+		encDec: encDec,
+	}
+}
+
+// Encode string.
+func (c *CustomEncoding) Encode(str string) ([]byte, error) {
```
Comment (goten4) [inline data/codings.go:65]: `c` has no need to be a pointer here and I think it's the same for all bellow

**Review by goten4 — COMMENTED** [review]
lgtm

File: data/codings.go (hunk 1 above)
Comment (linxGnu) [reply data/codings.go:65]: @goten4 the reason to use pointer for struct func is: prevent copy

**Comment by bachhh** [comment]
lgtm

**Review by bachhh — APPROVED** [review]

**Comment by linxGnu** [comment]
Thank you very much for reviewing @goten4 @bachhh

**Pull request merged**